orcoursetrion/config.py

```diff
@@ -13,7 +13,7 @@
     'ORC_GH_API_URL': 'https://github.mit.edu/api/v3/',
 
     # GitHub Committer Name
-    'ORC_GH_NAME': 'Orcoursetrion ODL',
+    'ORC_GH_NAME': 'Orcoursetrion',
 
     # GitHub Committer Email
     'ORC_GH_EMAIL': 'orcoursetrion@example.com',
```

Comment:
That makes it difficult to test and a bit less flexible, you can configure them with environment variables though